### PR TITLE
Update filedisk, partitioninfo and ext2fs

### PR DESCRIPTION
**Pull request by zvin**
---- Autogenerated Waffleboard Connection: Connects to #35

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,10 +19,18 @@
...
   },
   "scripts": {
-    "test": "gulp test && coffee tests/e2e.coffee",
-    "prepublish": "gulp build",
-    "readme": "jsdoc2md --template doc/README.hbs build/imagefs.js > README.md"
+    "lint": "resin-lint lib tests",
+    "build": "coffee --bare -o build lib",
+    "test": "npm run lint && npm run build && coffee tests/e2e.coffee",
+    "readme": "jsdoc2md --template doc/README.hbs build/imagefs.js > README.md",
+    "prepublish": "npm run test && npm run readme"
```
Comment (jhermsmeier) [inline package.json]: Should this also run the `build`?

Comment (zvin) [reply package.json]: build is already ran by test

Comment (zvin) [reply package.json]: @jhermsmeier I updated it so test no longer builds and prebublish tests, builds and creates the docs.

**Comment by zvin** [comment]
>Some JS files in build have been removed, but their coffee-script counterparts still exist – is this intentional?
@jhermsmeier yes, the whole `build` folder was removed

**Comment by jhermsmeier** [comment]
> the whole build folder was removed

Aye, the `build` dir should be added to the `.gitignore` then

**Comment by zvin** [comment]
>Aye, the build dir should be added to the .gitignore then
@jhermsmeier right, done

**Comment by zvin** [comment]
>LGTM, unless this isn't supposed to fail on Windows (?)

@jhermsmeier yeah the build should work, this is a problem with the latest node-pre-gyp in node-ext2fs.
I'll fix it before merging this.

**Pull request merged**